src/core/handleErrorFactory.js

```diff
@@ -12,9 +12,12 @@ governing permissions and limitations under the License.
 
 import { toError } from "../utils";
 
+const suppressionNote =
+  "Note: Errors can be suppressed via the errorsEnabled configuration option.";
+
```

Comment:
It may not be clear what you have to set the errorsEnabled configuration option in order to suppress errors.

`Note: Errors can be suppressed by setting errorsEnabled to false as an option to configure`

Reply:
I was hoping it would subtly prompt them to go read the documentation before just setting it to `false` so they could understand the ramifications a little more. I'll change it though.